Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem 'inspec', '1.51.6', require: false
+gem 'inspec', '3.0.9', require: false
```

Comment:
Need to revert this back to `1.51.6` in order to use aws-sdk 3 which is needed by inspec-aem during stack manager event checks.

Reply:
Unless it's need for amazon linux 2 support in `train`.